### PR TITLE
Body

### PR DESCRIPTION
**Pull request by SabirAmeen**
hi @lekshmi615213 
I have updated the code. please review it and suggest modifications.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+<body>
+    <header class="main-head">
+        <span class="menu">
+            <img id="menu-icon" src="images/menu.png" alt="Menu-btn">
+        </span>
+        <span class="logo">
+            <a href=""><img id="verge-logo" src="images/logo.png" alt="verge-logo"></a>
+        </span>
+        <div class="article-toggle">
```
Comment (GayathriMohan) [inline index.html]: Use span instead of div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+        <article id="important-art-three" class="important-art-common" onclick="location.href='';">
+            <h2 class="important-art-head">Laika Studios and the making of Kubo and the Two Strings</h2>
+            <span class="author">
+         			 By Jordan Golson
+         			 <span class="comment">
+         				19
+         			</span>
+            </span>
+        </article>
+        <section class="article-scroller" style="font-size:0;">
```
Comment (GayathriMohan) [inline index.html]: Remove style from html page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+						Lyft pushes back at Uber’s unsavory tactics
+						</a>
+					</h3>
+                <span class="entry-author"><a href="">James Vincent</a></span>
+                <em class="article-time">21 minutes ago</em>
+            </li>
+        </ul>
+    </section>
+    <footer>
+        <div id="back-to-top" onclick="location.href='';">
```
Comment (GayathriMohan) [inline index.html]: Use <a> only.

Comment (GayathriMohan) [inline index.html]: Use 'a href' instead of a complete div

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,718 @@
+body {
+    margin: 0;
```
Comment (GayathriMohan) [inline css/style.css]: margin and height already set to 0 in reset.css

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,718 @@
...
+    font-family: ff-din-web-condensed, Helvetica, Arial, sans-serif;
+}
+
+.hidden {
+    display: none;
+}
+
+.main-head {
+    height: 60px;
+    margin: 0;
```
Comment (GayathriMohan) [inline css/style.css]: Dont mention margin:0 if you are using reset.css

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,718 @@
...
+.nav-bar {
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    background-color: #000;
+    color: #FFF;
+    border-right: 2px solid #fa4b2a;
+    width: 240px;
+    display: none;
```
Comment (GayathriMohan) [inline css/style.css]: All the selectors with display none can be chained together

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,718 @@
...
+}
+
+.article-new {
+    padding: 0.8rem 0.8rem 0.8rem 0;
+}
+
+.article-list {
+    list-style: none;
+    margin: 0;
+    padding: 0 0 1rem 1rem;
```
Comment (GayathriMohan) [inline css/style.css]: Use padding-bottom and padding-left

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,718 @@
...
+    padding: .4rem .2rem .2rem;
+    font-weight: 700;
+    min-width: 20px;
+    text-align: center;
+}
+
+.comment:after {
+    content: "";
+    border-top-color: #000;
+    width: 0;
```
Comment (GayathriMohan) [inline css/style.css]: why setting to 0?

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,718 @@
...
+    right: 0;
+}
+
+#important-art-one {
+    background-image: url("https://cdn0.vox-cdn.com/thumbor/ivsHgXHY572hECkjL2fcMiZQfZs=/0x0:2039x1147/680x383/cdn0.vox-cdn.com/uploads/chorus_image/image/50439647/twitter-stock-0936.0.0.jpg");
+    width: 100%;
+}
+
+#important-art-one:after {
+    content: "";
```
Comment (GayathriMohan) [inline css/style.css]: Include all the common elements in :after in a class rather than repeating it every time.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,718 @@
...
+    display: inline-block;
+}
+
+#entry-mobile h3 {
+    margin: 0;
+    font-family: Adelle, Georgia, serif;
+    font-size: 1.2rem;
+    font-weight: 400;
+    margin: 0;
+    width: 200px;
```
Comment (GayathriMohan) [inline css/style.css]: 2 width values given .

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,718 @@
...
+
+#left-foot-nav {
+    list-style: none;
+    display: inline-block;
+    width: 50%;
+    padding: 0;
+}
+
+#left-foot-nav li {
+    padding: 0 0 0.5rem 1.5rem;
```
Comment (GayathriMohan) [inline css/style.css]: Use padding-bottom and padding-left

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,718 @@
...
+}
+
+.hidden {
+    display: none;
+}
+
+.main-head {
+    height: 60px;
+    margin: 0;
+    width: 100%;
```
Comment (anoopda) [inline css/style.css]: There is no need to mention 100% width .

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,718 @@
...
+    background-color: #f0f0f0;
+}
+
+.article-toggle {
+    margin-top: 5px;
+    float: right;
+    color: #FFF;
+    width: 5rem;
+    padding-right: 0.3rem;
+    padding-left: 0.1rem;
```
Comment (anoopda) [inline css/style.css]: use shorthands

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,718 @@
...
+
+#copyrights img {
+    width: 25px;
+    height: 25px;
+    transform: rotateZ(90deg);
+    margin-bottom: 8px;
+}
+
+#copyrights a {
+    color: #fa4b2a;
```
Comment (GayathriMohan) [inline css/style.css]: Use a class to include this color

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,718 @@
...
+    #foot-logo {
+        display: block;
+        width: 100%;
+        height: 100%;
+        margin-bottom: 10px;
+    }
+    #search-foot {
+        width: 90%;
+    }
+    #left-foot-nav {
```
Comment (GayathriMohan) [inline css/style.css]: 2  elements with same width. Apply chaining.

File: css/style.css [hunk 16]
```diff
@@ -0,0 +1,718 @@
...
+    }
+}
+
+@media screen and (min-width: 700px) {
+    #important-art-one {
+        height: 300px;
+        width: 50%;
+        float: left;
+    }
+    #important-art-two {
```
Comment (GayathriMohan) [inline css/style.css]: #important-art-two and #important-art-three repetition

File: css/style.css [hunk 17]
```diff
@@ -0,0 +1,718 @@
...
+    .entry-section-left {
+        border-bottom: 1px solid #ddd;
+        padding: 2rem 1rem;
+    }
+    #entry-center {
+        display: inline-block;
+        width: 50%;
+        box-sizing: border-box;
+    }
+    #entry-center a {
```
Comment (GayathriMohan) [inline css/style.css]: Chain it with #entry-left a
Chain #entry-center h3 with #entry-left h3

File: css/style.css [hunk 18]
```diff
@@ -0,0 +1,718 @@
...
+    width: 100%;
+    height: 100%;
+    background-image: linear-gradient(135deg, #417711, #dc4225);
+    opacity: 0.7;
+}
+
+#entry-mobile {
+    padding: 0;
+    width: 100%;
+    list-style: none;
```
Comment (anoopda) [inline css/style.css]: no need of list-style:none, if padding is 0

File: css/style.css [hunk 19]
```diff
@@ -0,0 +1,718 @@
...
+    width: 50%;
+    padding: 0;
+}
+
+#left-foot-nav li {
+    padding: 0 0 0.5rem 1.5rem;
+}
+
+#center-foot-nav {
+    list-style: none;
```
Comment (anoopda) [inline css/style.css]: list type and paddding

**Pull request closed without merging**